### ISSUE TITLE
`be spoom tc` -> undefined method `cyan' for "i":String

### ISSUE THREAD

**Issue by ekampp** [Shopify/spoom#113]
**Describe the bug**

After installing tapioca and following their [migrating to tapioca guide](https://github.com/Shopify/tapioca/wiki/Migrating-to-Tapioca) (specifically [this step using Spoom](https://github.com/Shopify/tapioca/wiki/Migrating-to-Tapioca#bump-strictness-on-files)) I get this error when running `bundle exec spoom tc`:

```
(...)/versions/3.0.2/lib/ruby/gems/3.0.0/gems/spoom-1.1.5/lib/spoom/cli/run.rb:102:in `block in colorize_message': undefined method `cyan' for "i":String (NoMethodError)
```

**To Reproduce**

```
bundle info spoom
  * spoom (1.1.5)
	Summary: Useful tools for Sorbet enthusiasts.
	Homepage: https://github.com/Shopify/spoom
	Path: (...)/versions/3.0.2/lib/ruby/gems/3.0.0/gems/spoom-1.1.5
```

1. Installing tapioca and following their [migrating to tapioca guide](https://github.com/Shopify/tapioca/wiki/Migrating-to-Tapioca)
2. [Bump files to typed](https://github.com/Shopify/tapioca/wiki/Migrating-to-Tapioca#bump-strictness-on-files)
3. Run `bundle exec spoom tc`
4. See errror in console

**Expected behavior**

I would expect it to list the reasons why it didn't auto-bump my files.

**Comment by Morriar** [comment]
👋 Hey Emil,

I believe this has been fixed in https://github.com/Shopify/spoom/pull/110.

Feel free to reopen if it's not the case.

Thanks for the your report.

**Comment by ekampp** [comment]
Hi, @Morriar 👋🏼 

Thanks for taking the time to help me! It's really appreciated. It does look like the same bug. However, Running `bundle update spoom` doesn't upgrade to a version of spoom that has that PR's code included. 

What does the release schedule look like? Or should I grab it from `github.com/Shopify/spoom` in my gemfile? 

Thanks!

**Comment by Morriar** [comment]
I just released [Spoom 1.1.6](https://rubygems.org/gems/spoom/versions/1.1.6) with the patch. Let me know if the problem persists.
